test_script.sh

```diff
@@ -0,0 +1 @@
+./gradlew clean check test
```

Comment:
We have not used the unit test support here. Our tests need the android api's. So we need to use 'connectedCheck'